config/environments/test.rb

```diff
@@ -36,9 +36,10 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Store uploaded files on the local file system in a temporary directory.
+  # Store uploaded files in the db for testing
   config.active_storage.service = :test
-  config.active_storage.variant_processor = :mini_magick # vips is better but does not work on Mac Apple Silicon so using this in dev & test
+  # vips is better but does not work on Mac Apple Silicon so using this in dev & test
+  config.active_storage.variant_processor = :mini_magick
```

Comment:
I think it’s worth reverting this. Even though it makes the line really long, which is a bit annoying, the few times I do add comments I try to bias towards having them on the actual line they’re explaining rather than their own line, only because it slightly decreases the frequency that someone updates a line without reading the comment.

I feel like I stated that in a confusing way. All I’m trying to say is: if the comment is on its own line, it’s more likely to get out of sync. So let’s keep it connected to the mini_magick thing even though it makes the line really long.